Examples/CMakeLists.txt

```diff
@@ -17,17 +17,6 @@ endif()
 option(SimpleITK_TESTING_NOSHOW "Disable the Show function in SimpleITK testing." ON)
 mark_as_advanced(SimpleITK_TESTING_NOSHOW)
 
-
-if(SimpleITK_USE_ELASTIX)
```

Comment:
The subdirectory should be conditionally included.

Reply:
Done.